charts/ecosystem/templates/event-streams-secret.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.eventStreamsSecretName }}
```

Comment:
Potential issue - if we don't provide the name of a secret, a token won't be generated, which might cause issues when the other deployments try loading the token. Maybe we could generate a name if this happens?